### DISCUSSION TITLE
Run calibration issues

### DISCUSSION THREAD

**Discussion by VeronicaYumiceba** [Katsevich-Lab/sceptre#122] [Q&A] [answered]
Hi, 
I am currently using sceptre version 9.1 and while running the calibration I encountered an error (see below) and the calibration is not performed.

Running calibration_check in parallel. Change directories to /tmp/RtmpR92zW5/sceptre_logs/ and view the files calibration_check_*.out for progress updates.  ✓
Error in data.table::rbindlist(res, fill = TRUE) : 
  Item 1 of input is not a data.frame, data.table or list
In addition: Warning messages:
1: In parallel::mclapply(seq_along(partitioned_response_ids), function(proc_id) run_precomp_on_given_responses(partitioned_response_ids[[proc_id]],  :
  scheduled cores 1, 2, 3, 5, 6, 8, 10, 11, 14, 15, 19, 20, 23, 25, 26, 27, 28, 31, 32, 33, 36, 37, 40, 44, 45, 48 did not deliver results, all values of the jobs will be affected
2: In parallel::mclapply(seq_along(partitioned_grna_group_ids), function(proc_id) perform_association_analysis(partitioned_grna_group_ids[[proc_id]],  :
  all scheduled cores encountered errors in user code

**Comment by timothy-barry** [comment, marked as answer]
Hi Veronica,

Two things:
1. This seems like more of a bug report than a discussion question, so I created an issue for this post. We can continue the conversation over [there](https://github.com/Katsevich-Lab/sceptre/issues/123).
2. It is going to be challenging for me to debug this without access to the data. Would you mind sending your `sceptre_object` and the script that you are using to analyze your `sceptre_object` to my email (tbarry@hsph.harvard.edu)? This way I can take a closer look.
